spec/factories/other_assets_declarations.rb

```diff
@@ -45,10 +45,10 @@
     end
 
     trait :mix_of_values do
-      second_home_value { Faker::Number.decimal(6, 2) }
-      second_home_mortgage { Faker::Number.decimal(6, 2) }
-      second_home_percentage { Faker::Number.decimal(6, 2) }
-      timeshare_property_value { Faker::Number.decimal(6, 2) }
+      second_home_value { rand(1...1_000_000.0).round(2) }
+      second_home_mortgage { rand(1...1_000_000.0).round(2) }
+      second_home_percentage { rand(1...1_000_000.0).round(2) }
```

Comment:
As this is a percentage field I think it needs to be `second_home_percentage { rand(1...99.0).round(2) }`.

Reply:
Good spot. I missed this one